Tutorial HTML/Html 01/README.md

```diff
@@ -0,0 +1,85 @@
+# Aplicatii web pentru incepatori
+######  Autor : Oancea Ionut Eugen
+
+![HACKED.io: Hands on FirefoxOS](https://arcturus.github.io/firefoxos-hackedio/img/html5_css_javascript.png)
+
+Orice aplicatie web acesibila intr-un browser foloseste o combinatie de 3 limbaje pentru a ajunge la forma sa finala (HTML + CSS + JavaScript). 
+Pentru a continua ne este indeajuns sa intelegem pe scurt rolul fiecarui limbaj intr-o aplicatie
+Nu ne vom arunca prea mult inca in detalii la acest punct.
+
+- *Am adaugat un link catre wikipedia cu niste explicatii, pentru cei care doresc insa acest lucru.*
+
+#### - HTML ( Hyper text markup language )
+[HTML Wikipedia](https://ro.wikipedia.org/wiki/HyperText_Markup_Language)
+Este folosit pentru a crea pagini web simple ce pot fi afișate într-un browser.
+Scopul HTML este  in general prezentarea informațiilor – paragrafe fonturi, tabele ș.a.m.d si este limitat la acest lucru de unul singur.
+
+#### - CSS ( Cascading Style Sheets )
+[CSS Wikipedia](https://ro.wikipedia.org/wiki/Cascading_Style_Sheets)
+ CSS formateaza HTML-ul scris, si cu ajutorul unui set de prorietati se pot pozitiona elemente in pagina, se pot face animatii, ș.a.m.d
+
+#### - JS (JavaScript)
+ ##### A nu se confunda cu Java ("Java is to JavaScript what car is to carpet")
+
+[JS Wikipedia](https://ro.wikipedia.org/wiki/JavaScript)
+ Este un limbaj de programare ce poate fi folosit pentru realizarea logicii unei pagini web ( formulare autentificare, ce se intampla cand apesi un buton, ș.a.m.d) precum si diverse aplicatii, jocuri, animatii  ș.a.m.d.
+
+Aceste 3 lucruri combinate pot avea ca rezultat o [aplicatie web](https://ro.wikipedia.org/wiki/Aplica%C8%9Bie_web)
+
+## HTML
+
+O aplicatie web nu poate functiona fara macar o boaba de HTML asa ca vom incepe cu acest lucru.
+
+- Sa presupunem ca dorim sa creeam o pagina web care sa contina poezii
+Avem asignata cate o poezie pentru fiecare pagina, si deasemenea dorim sa afisam un tabel cu date despre fiecare poezie pe pagina ei
+Deasemenea vom dori ca aceasta pagina web sa contina o sectiune ce va permite navigarea intre poezii,
+
+HTML este un limbaj de interpretare.
+Acest lucru ne spune ca HTML e un format de text cu anumite componente ce sunt interpretate de browser pentru a fi afisate in mod corespunzator.
+
+De exemplu ne putem gandi aici la relatia dintre un cititor si un poet.
+Poetul scrie o poezie de sucess cu gandul la dragoste, si fericire, dintr-o experienta trecuta.
+Vine citirorul, se apuca de citit si datorita experientelor sale diferite acesta interpreteaza poezia ca fiind una trista, si o povesteste mai departe ca atare.
+In cazul de fata cititorul nostru este browserul, iar scriitorul fiind cel care creaza o pagina web
```

Comment:
Putin confuz exemplul. In principiu o pagina Web care respecta standardul W3C ar trebui sa arate similar in toate browserele. Nu prea inteleg de ce cititorul o interpreteaza diferit, adica ca fiind trista...?

Reply:
Vroiam sa ilustrez idea de interpretare ca fiind un lucru nu tocmai tehnic, ci ceva obiectiv.
Chestia cu cititorul poate interpreta chestia ca fiind trista cand ea poate e fericita inseamna c abrowserul vede cu totul altceva
Tu vezi <a href="ceva">Hello</a> dar defapt browserul vede un text pe care il va afisa albastru by default ce te redirectioneaza la ceva cand dai click pe el.
Am sa ma gandesc mai bine aici